blockchain/types/tx_internal_data.go

```diff
@@ -103,6 +106,10 @@ var (
 	errValueKeyDataMustByteSlice         = errors.New("Data must be a slice of bytes")
 	errValueKeyFeeRatioMustUint8         = errors.New("FeeRatio must be a type of uint8")
 	errValueKeyCodeFormatInvalid         = errors.New("The smart contract code format is invalid")
+	errValueKeyAccessListInvalid         = errors.New("AccessList must be a type of AccessList")
+	errValueKeyChainIDInvalid            = errors.New("ChainID must be a type of ChainID")
+
+	ErrTxTypeNotSupported = errors.New("transaction type not supported")
```

Comment:
```suggestion
	errTxTypeNotSupported = errors.New("transaction type not supported")
```
How about change error name to use lowercase?
Or maybe we can use `errUndefinedTxType`?

Reply:
This error used  another package, so it must set a uppercase.